src/effects/effectchainslot.cpp

```diff
@@ -44,6 +44,10 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
     m_pControlChainSuperParameter->set(0.0);
     m_pControlChainSuperParameter->setDefaultValue(0.0);
 
+    m_pControlChainSuperForceUpdate = new ControlObject(ConfigKey(m_group, "super_force_next_update"));
```

Comment:
This is set to 1 to set the state of Mixxx to match the state of a controller on startup. Otherwise the normal soft takeover between the super/metaknob and parameters creates a disconnect between the position of the knob and how the effect sounds. I will document this on the wiki when merged. I have implemented this in the [JS library](https://github.com/mixxxdj/mixxx/pull/1054/files#diff-400922cb0109e689c3d5404810aec4dfR921).